### PR TITLE
Окно регистрации и авторизации

### PR DESCRIPTION
**Pull request by NikitaGoroshko7**
https://trello.com/c/3DfBOrjm/9-%D0%BE%D0%BA%D0%BD%D0%BE-%D1%80%D0%B5%D0%B3%D0%B8%D1%81%D1%82%D1%80%D0%B0%D1%86%D0%B8%D0%B8-%D0%B8-%D0%B0%D0%B2%D1%82%D0%BE%D1%80%D0%B8%D0%B7%D0%B0%D1%86%D0%B8%D0%B8
Added AccountController and Views Login and Register.
Added style for Views

### CODE REVIEW

File: CrowdFundingApplication/obj/CrowdFundingApplication.csproj.nuget.dgspec.json [hunk 1]
```diff
@@ -1,23 +1,23 @@
 {
   "format": 1,
   "restore": {
-    "E:\\Project\\CrowdFundingApplication\\CrowdFundingApplication.csproj": {}
+    "E:\\Программирование\\!Диплом\\CrowdFunding Platform\\CrowdFundingApplication\\CrowdFundingApplication.csproj": {}
```
Comment (GalayM) [inline CrowdFundingApplication/obj/CrowdFundingApplication.csproj.nuget.dgspec.json]: Why doesn't gitignore file work?

File: CrowdFundingApplication/wwwroot/css/LogAndAutho.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+.btn-primary {
+    background-color: #a30846;
+    border-color: #a30846;
+    border-radius: 8px;
+}
+
+    .btn-primary:active {
+        background-color: #a30846 !important;
```
Comment (GalayM) [inline CrowdFundingApplication/wwwroot/css/LogAndAutho.css:44]: It is better to override this for all buttons in a common style for the entire application.

File: CrowdFundingApplication/Views/Account/Login.cshtml [hunk 3]
```diff
@@ -0,0 +1,66 @@
+﻿@{
+    Layout = null;
+}
+
+<html lang="en">
```
Comment (GalayM) [inline CrowdFundingApplication/Views/Account/Login.cshtml:5]: Please use layout to pages

File: CrowdFundingApplication/obj/CrowdFundingApplication.csproj.nuget.dgspec.json (hunk 1 above)
Comment (NikitaGoroshko7) [reply CrowdFundingApplication/obj/CrowdFundingApplication.csproj.nuget.dgspec.json]: gitignore already working

**Pull request merged**